src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -1884,45 +1877,39 @@ subroutine ocn_reconstruct_gm_vectors(diagnosticsPool, meshPool) !{{{
       call mpas_pool_get_array(diagnosticsPool, 'GMBolusVelocityZonal', GMBolusVelocityZonal)
       call mpas_pool_get_array(diagnosticsPool, 'GMBolusVelocityMeridional', GMBolusVelocityMeridional)
 
-      call mpas_pool_get_array(diagnosticsPool, 'relativeSlopeTopOfEdge', relativeSlopeTopOfEdge)
-      call mpas_pool_get_array(diagnosticsPool, 'relativeSlopeTopOfCellX', relativeSlopeTopOfCellX)
-      call mpas_pool_get_array(diagnosticsPool, 'relativeSlopeTopOfCellY', relativeSlopeTopOfCellY)
-      call mpas_pool_get_array(diagnosticsPool, 'relativeSlopeTopOfCellZ', relativeSlopeTopOfCellZ)
-      call mpas_pool_get_array(diagnosticsPool, 'relativeSlopeTopOfCellZonal', relativeSlopeTopOfCellZonal)
-      call mpas_pool_get_array(diagnosticsPool, 'relativeSlopeTopOfCellMeridional', relativeSlopeTopOfCellMeridional)
-
       call mpas_pool_get_array(diagnosticsPool, 'gmStreamFuncTopOfEdge', gmStreamFuncTopOfEdge)
       call mpas_pool_get_array(diagnosticsPool, 'GMStreamFuncX', GMStreamFuncX)
       call mpas_pool_get_array(diagnosticsPool, 'GMStreamFuncY', GMStreamFuncY)
       call mpas_pool_get_array(diagnosticsPool, 'GMStreamFuncZ', GMStreamFuncZ)
       call mpas_pool_get_array(diagnosticsPool, 'GMStreamFuncZonal', GMStreamFuncZonal)
       call mpas_pool_get_array(diagnosticsPool, 'GMStreamFuncMeridional', GMStreamFuncMeridional)
 
-      call mpas_reconstruct(meshPool, normalTransportVelocity,          &
+      call mpas_pool_get_array(diagnosticsPool, 'slopeTriadUp', slopeTriadUp)
+      call mpas_pool_get_array(diagnosticsPool, 'slopeTriadDown', slopeTriadDown)
+
+      ! mrp temporary change to see slope
+      !call mpas_reconstruct(meshPool, normalTransportVelocity,          &
+      call mpas_reconstruct(meshPool, slopeTriadUp(:,1,:),          &
                        transportVelocityX,            &
                        transportVelocityY,            &
                        transportVelocityZ,            &
                        transportVelocityZonal,        &
                        transportVelocityMeridional    &
                       )
 
-      call mpas_reconstruct(meshPool, normalGMBolusVelocity,          &
+      ! mrp temporary change to see slope
+      !call mpas_reconstruct(meshPool, normalGMBolusVelocity,          &
+      call mpas_reconstruct(meshPool, slopeTriadUp(:,2,:),          &
```

Comment:
Should these be reverted as well?  We would like to output the bolus velocity again.

Reply:
Thanks for checking, that was a mistake. I'm reverting those.